utils/duplicate/archiver.py

```diff
@@ -0,0 +1,91 @@
+import logging
+from copy import deepcopy
+from typing import Tuple
+
+from requests.exceptions import HTTPError
+from assertpy import assert_warn
+from biosamples_v4.api import Client as BioSamplesClient
+from ingest.api.ingestapi import IngestApi
+from submission_broker.services.biosamples import BioSamples
+
+from converter.biosamples import BioSamplesConverter
+from .loader import DuplicateLoader
+from .submission import DuplicateSubmission, HcaSubmission
+
+
+class DuplicateArchiver:
```

Comment:
What does "Duplicate" mean?

Reply:
synonyms: clone, copy, create existing as new. Hard thing to name.
Any suggestions?